io/net/virt-net/network_virtualization.py

```diff
@@ -271,22 +310,31 @@ def test_clientfailover(self):
         except CmdError as details:
             self.log.debug(str(details))
             self.fail("Client initiated Failover for Network virtualized \
-                       device has failed")
+                      device has failed")
 
     def test_backingdevremove(self):
         '''
         Removing Backing device for Network virtualized device
         '''
+        if self.check_slot_availability():
+            self.fail("Slot does not exist")
+        self.update_backing_devices()
         pre_remove = self.backing_dev_count()
-        self.backing_dev_add_remove('remove')
+        for count in range(1, self.backingdev_count):
+            self.backing_dev_add_remove('remove', count)
         post_remove = self.backing_dev_count()
-        if pre_remove - post_remove != 1:
+        if pre_remove - post_remove + 1 != self.backingdev_count:
+            self.log.debug(pre_remove - post_remove + 1)
+            self.log.debug(self.backingdev_count)
```

Comment:
can these statement merge in single line 
self.log.debug("slot after remove %s and device count % " %((pre_remove - post_remove + 1),  self.backingdev_count)

Reply:
variable usage, I agree. Will this matter that much ?

Reply:
not very hard on this only just a suggestion

Reply:
sure, will think about it